### PR TITLE
Add Zyxel cfg nat addr map get

### PR DESCRIPTION
**Pull request by elavaud**
##### ISSUE TYPE
 - New Template Pull Request

##### COMPONENT
Zyxel OS, cfg nat addr map get

### CODE REVIEW

File: ntc_templates/templates/zyxel_os_cfg_nat_addr_map_get.textfsm [hunk 1]
```diff
@@ -0,0 +1,19 @@
+Value INDEX (\d+)
+Value NAME (.+?)
+Value INTERFACE (\w+)
+Value TYPE (\S+?)
+Value LOCAL_START_IP ((\d+\.\d+\.\d+\.\d+)?)
```
Comment (itdependsnetworks) [inline ntc_templates/templates/zyxel_os_cfg_nat_addr_map_get.textfsm:5]: I don't think this pattern will actually work if the IPs are optional. LOCAL_START_IP will get conflated for LOCAL_END_IP as an example. If this actually exists in the real world, then would need tests, and likely another strategy to attack.

Comment (elavaud) [reply ntc_templates/templates/zyxel_os_cfg_nat_addr_map_get.textfsm:5]: Indeed. It should not be optional. Will correct

Comment (itdependsnetworks) [reply ntc_templates/templates/zyxel_os_cfg_nat_addr_map_get.textfsm]: I only commented on the one, but I meant it for all of them :)

Comment (elavaud) [reply ntc_templates/templates/zyxel_os_cfg_nat_addr_map_get.textfsm]: Ok, I fine tuned adding some tests with raw data I collected directly after manually configuring the address 
mapping through the UI. Let me know.
Sorry for the troubles. I am adding old work of a former colleague and I am new to networking.

**Review by jmcgill298 — COMMENTED** [review]
needs rebase

**Comment by elavaud** [comment]
Grouped in #1142

**Pull request closed without merging**